### DISCUSSION TITLE
Using Luxon on cast functions

### DISCUSSION THREAD

**Discussion by spampibridge** [mia-platform/community#120] [Q&A] [answered]
Hi, 

i have to parse a string formatted date without timezone and convert in a Date js object.
I want to do this task on my custom cast-function.

This task can be easily accomplished using Luxon js lib.

Can i use Luxon library in my cast-function?

Thanks.

**Comment by ThisIsDemetrio** [comment, marked as answer]
Hi @spampibridge! 

Unfortunately, right now it's not possible to use external libraries to handle dates in cast functions.
According to the [docs](https://docs.mia-platform.eu/docs/fast_data/configuration/cast_functions#technical-limitation), only lodash, mongodb and radma are available.

But I think that Luxon (or a similar lib, like date-fns or dayjs) could be really useful for simplify functions that involves dates (and timezone, especially), we might think to introduce it in the future.

Hope it helped,
|D|
